src/Search/Gearman/Command/QueueCommand.php

```diff
@@ -120,23 +125,29 @@ public function checkStatus(InputInterface $input, LoggerInterface $logger)
 
     public function loop(InputInterface $input, LoggerInterface $logger)
     {
+        $logger->info('Loop start... [');
         $topic = $input->getArgument('topic');
         $timeout = $input->getOption('queue-timeout');
-        $logger->info('Hello queue. topic: '.$topic);
+        $logger->info('-> Hello queue. topic: '.$topic);
```

Comment:
Okay changed this file to use that, I imagine when running verbose it will show the details